engine/classes/ElggPlugin.php

```diff
@@ -871,7 +871,7 @@ protected function includeFile($filename) {
 		}
 
 		try {
-			$ret = Includer::includeFile($filepath);
+			$ret = Application::requireSetupFileOnce($filepath);
```

Comment:
this will fail (without exception) and crash the system if the `$filepath` isn't present.

Reply:
Doesn't it check if filepath is readable above?

Reply:
> Doesn't it check if filepath is readable above?

correct